cpp/src/transpose/transpose.cu

```diff
@@ -89,7 +92,11 @@ void gpu_transpose_valids(gdf_valid_type **in_cols_valid,
         int num_nulls = __popc(active_threads) - __popc(result_mask);
         atomicAdd(out_cols_null_count + j, num_nulls);
       }
+      
+      j += stride_y;
```

Comment:
I believe `activate_threads` needs to be updated after every stride in the `y` dimension, because after every stride you introduce potential for warp divergence.

Reply:
warps are arranged in x direction. so in case one warp goes over the ncol limit then it has partially active threads. on the other hand, in the y direction, the whole warp is going to be inactive.
<img width="486" alt="screen shot 2019-02-12 at 3 10 48 am" src="https://user-images.githubusercontent.com/3027195/52595429-de777c00-2e73-11e9-86da-c32b0d13ce60.png">

Reply:
Indeed, I was mistaken. Good change.